.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Summary
+
```

Comment:
Add some description in these field for user to edit (noted to remove if unnecessary)